Sources/Core/Formatter/MultilineFormatter.swift

```diff
@@ -6,44 +6,50 @@
 //
 
 class MultilineFormatter: PrettyFormatter {
-    private let option: Debug.Option
+    private let indentSize: Int
 
-    init(option: Debug.Option) {
-        self.option = option
+    init(indentSize: Int) {
```

Comment:
Refactor: rename to `indentSize` from `indent` because the name `indent` is ambiguous.